examples/blog-update/mkdocs.yml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2016-2023 Martin Donath <martin.donath@squidfunk.com>
+#                         Alex Voss <alex@corealisation.com>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+# -----------------------------------------------------------------------------
+# Recommended: set up configuration validation, see https://t.ly/xpZXU
+# -----------------------------------------------------------------------------
+
+# Project information
+site_name: Blog that shows post updates
+
+# Theme
+theme:
+  name: material
+  custom_dir: overrides
+
+watch:
+  - overrides
```

Comment:
That is not necessary, I think. IMHO `mkdocs serve --watch-theme` is what you want.

Reply:
That is not how I read the documentation. That would not work if people did not use the name `overrides`. I assumed it meant the files in the theme, wherever it is resolved from but *not* overrides, useful that might be, though. In any case, would it not be easier to specify this in `mkdocs.yml` rather than having to remember doing it on the command-line? That is what I have always done.

Reply:
MkDocs will add the`custom_dir` directory to `config.theme.dirs`, [which are all watched when using `--watch-theme`](https://github.com/mkdocs/mkdocs/blob/828f4685f29dd9e986f18306d58d1cb383d00222/mkdocs/commands/serve.py#L94-L96).

Reply:
That is good to know. Still leaves the question whether it is a good idea to add it to the watchlist in `mkdocs.yml` to ensure it is watched even when people forget to add the `--watch-theme` part. Is there any reason why doing the latter would be better? My way seems (naively) safer.

Reply:
Good point, I forgot about `--watch-theme` as I always use it. We should leave it in `mkdocs.yml`.